package.json

```diff
@@ -1,19 +1,26 @@
 {
-  "name": "@tv-kitchen/appliance-template",
+  "name": "root",
+  "private": true,
   "version": "1.0.0",
   "description": "A template for TV Kitchen Appliances.",
```

Comment:
Whoops, I suppose we should've updated this description, too. 😬

Reply:
............. hotfix time...........